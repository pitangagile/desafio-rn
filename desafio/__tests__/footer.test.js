import React from 'react';
import Footer from '../src/components/footer';

import renderer from 'react-test-renderer';

test('Footer Snapshot', () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
