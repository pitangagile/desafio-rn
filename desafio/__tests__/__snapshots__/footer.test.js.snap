// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot 1`] = `
<View
  style={
    Object {
      "paddingVertical": 20,
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#999999"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
